tests/test_build_latex.py

```diff
@@ -1383,3 +1383,19 @@ def test_default_latex_documents():
     expected = [('index', 'stasi.tex', 'STASI™ Documentation',
                  r"Wolfgang Schäuble \& G'Beckstein.\@{}", 'manual')]
     assert default_latex_documents(config) == expected
+
+
+@skip_if_requested
+@skip_if_stylefiles_notfound
+@pytest.mark.sphinx('latex', testroot='latex-includegraphics',
+    confoverrides={'latex_documents': [
+                       ('index', 'python.tex', 'Test PDF build',
+                        'Georg Brandl', 'howto')
+                   ]})
+def test_includegraphics_oversized(app, status, warning):
+    app.builder.build_all()
+    print(status.getvalue())
+    print(warning.getvalue())
+    compile_latex_document(app)
+    result = (app.outdir / 'pdflatex/python-dimensions.txt').text(encoding='utf8')
```

Comment:
I would use SphinxTests-dimensions.txt on 1.8 branch. Oh I see now that I had initially here 

```
compile_latex_document(app, 'python.tex')
```

but I dropped the second argument when rebasing for testing on 1.8 and forgot to put it back when moving again to master.

Anyway this should work, but the pdflatex build is skipped as all others.